lib/matplotlib/tests/test_ticker.py

```diff
@@ -116,6 +114,31 @@ def test_low_number_of_majorticks(
         ax.xaxis.set_minor_locator(mticker.AutoMinorLocator())
         assert len(ax.xaxis.get_minorticklocs()) == expected_nb_minorticks
 
+    majorstep_minordivisions = [
+        (1, 5),
+        (2, 4),
+        (2.5, 5),
+        (5, 5),
+        (10, 5)
+    ]
+
+    # make sure we're checking all major steps in ticker.AutoLocator
+    majorsteps = [x[0] for x in majorstep_minordivisions]
+    assert np.allclose(majorsteps, mticker.AutoLocator()._steps)
```

Comment:
I wanted to make sure that the major steps in `majorstep_minordivisions` are the ones in AutoLocator()._steps with the goal  that `test_number_of_minor_ticks` will fail if that's not true. So I added this assert *outside* any test function. 
But if this assert fails pytest emits an error in the collection phase and doesn't run any tests.
Any suggestions how to improve this?

Reply:
Just move it to its own test with the comment that it's really intended to test that the next test is correctly parametrized.

Reply:
Done - thanks. 
I created a staticmethod, because it's the only way I found for the parameters to be accessible both in a test and by the ```@pytest.mark.parametrize``` decorator.

Reply:
Just make it a class variable like `params` just above?

Reply:
OK - done.